code/game/machinery/computer/security.dm

```diff
@@ -167,9 +167,17 @@
 				if(active2)
 					var/their_name = active2.fields["name"]
 					var/their_rank = active2.fields["rank"]
-					var/t1 = copytext(trim(sanitize(input("Enter Reason:", "Secure. records", null, null) as text)), 1, MAX_MESSAGE_LEN)
-					if(!t1)
+					var/t1
+					if(temp_href[2] == "execute")
+						t1 = copytext(trim(sanitize(input("Explain why they are being executed. Include a list of their crimes, and victims.", "EXECUTION ORDER", null, null) as text)), 1, MAX_MESSAGE_LEN)
+					else
+						t1 = copytext(trim(sanitize(input("Enter Reason:", "Secure. records", null, null) as text)), 1, MAX_MESSAGE_LEN)
+					var visible_reason

```

Comment:
Vars should have slashes, not spaces (I will never understand why BYOND even supports this syntax)